include/exadg/utilities/enum_utilities.h

```diff
@@ -19,39 +19,39 @@
  *  ______________________________________________________________________
  */
 
-// deal.II
-#include <deal.II/base/exceptions.h>
+#ifndef INCLUDE_EXADG_UTILITIES_ENUM_UTILITIES_H_
+#define INCLUDE_EXADG_UTILITIES_ENUM_UTILITIES_H_
+
+#include <magic_enum/magic_enum.hpp>
 
-// ExaDG
-#include <exadg/solvers_and_preconditioners/preconditioners/enum_types.h>
+#include <deal.II/base/exceptions.h>
 
 namespace ExaDG
 {
-namespace Elementwise
+/**
+ * Collection of enums utilities.
+ */
+namespace Utilities
```

Comment:
I like that you have wrapped these functions. This should allow to easily change the code in case this enum functionality becomes part of `std` some time in the future.